focus-blocked/index.html

```diff
@@ -96,64 +96,76 @@ <h1 id="focusTitle"></h1>
+          ? "This site is always blocked"
+          : block_type.includes("morning")
           ? "Blocked because you're doing your morning routine"
-          : block_type.includes('evening')
+          : block_type.includes("evening")
           ? "Blocked because you're doing your evening routine"
-          : '';
+          : "";
 
       const focus_tip = `<div class='notice-wrapper'>
-                          ${block_type ? '' : (`<a href=${current_url}>Click here to re-open the original URL ${
-        old_url ?? ''
-      }</a>`)}
+                          ${
+                            block_type
+                              ? ""
+                              : `<a href=${current_url}>Click here to re-open the original URL ${
+                                  old_url ?? ""
+                                }</a>`
+                          }
                           <h6>Want to reprogram your brain so you stay on task during focus blocks?
                             <a href="https://journals.sagepub.com/doi/abs/10.1177/1539449219876877?journalCode=otjb&">Research into breaking bad habits</a> indicates that if you take a positive action immediately after doing a bad habit, you'll start to disrupt the neural triggers for the bad habit. For example, you just opened ${old_url} when you were intending to focus${
         focus_mode ? ` on ${focus_mode}` : ``
       }. Try doing some deep breathing for 5 seconds, get up and stretch or go grab a glass of water. <a href="https://journals.sagepub.com/doi/full/10.1177/1539449219876877">Fritz et al's study</a> indicates this approach works much better than punishing yourself (no need to give yourself a mild electric shock for being so naughty). If you want a primer on the science of habit formation, <a href="https://hubermanlab.com/the-science-of-making-and-breaking-habits/">episode 53 of the Huberman Lab podcast</a> is worth a listen.</div>
                           </h6>`;
 
       if (block_type) {
-        document.getElementById('focusTitle').innerText = blocked_message;
-        document.getElementById(
-          'progressWrapper'
-        ).innerHTML = `<div class='notice-wrapper'><h6 class='centeredText'>Back to your plans for world domination! Save ${old_url} for when you've finished boiling the oceans.</h6></div>`;
-        document.getElementById(
-          'focusTipWrapper'
-        ).innerHTML = `<a id='showFocusTip'>Get a tip for staying focused</a>`;
-        document.getElementById('showFocusTip').onclick = function () {
-          document.getElementById('focusTipWrapper').innerHTML = focus_tip;
-        };
+        document.getElementById("focusTitle").innerText = blocked_message;
+        if (block_type === "always-block") {
+          document.getElementById(
+            "progressWrapper"
+          ).innerHTML = `<div class='notice-wrapper'><h6 class='centeredText'>If you want to allow ${old_url}, go to Preferences > Always Blocked URLs</h6></div>`;
```

Comment:
Add the image I shared.

Make it "{old_url} is configured to be always blocked. If you want to allow ${old_url}, go to Preferences > Always Blocked URLs"